Haskell/Haskell.sublime-syntax

```diff
@@ -331,7 +331,7 @@ contexts:
   comment:
     # As of build 4079 the [:punct:] class is missing some ASCII chars that
     # Unicode considers part of the Symbol category.
-    - match: '--(?![[:punct:]!-/:-@\[-`{-~])'
+    - match: '--*(?![[:punct:]!-/:-@\[-`{-~])'
```

Comment:
```suggestion
    - match: '--+(?![[:punct:]!-/:-@\[-`{-~])'
```

Reply:
Ah, good catch; we want *1* or more. Thank you :+1: